src/main/java/com/groupbyinc/quickstart/controller/NavigationController.java

```
@@ -251,8 +263,23 @@ protected ModelAndView handleSearch(HttpServletRequest request, HttpServletRespo
                 query.setBiasingProfile(profile);
             }
 
-            // pass the raw json representation of the query into the view regardless of errors
-            model.put("rawQuery" + i, query.setReturnBinary(false).getBridgeJson(clientKey));
+			String strategy = "[{ 'terms': 2, 'mustMatch': 2 }, { 'terms': 3, 'mustMatch': 2 }, { 'terms': 4, 'mustMatch': 3 }, { 'terms': 5, 'mustMatch': 3 }, { 'terms': 6, 'mustMatch': 4 }, { 'terms': 7, 'mustMatch': 4 }, { 'terms': 8, 'mustMatch': 5 }, { 'termsGreaterThan': 8, 'mustMatch': 60, 'percentage': true }]";
+			try {
+				strategy = matchStrategies[i].trim();
+				MatchStrategy oMatchStrategy = Utils.getMatchStrategy(strategy);
+				if (oMatchStrategy != null) {
+					query.setMatchStrategy(oMatchStrategy);
+				}
+
+			} catch (Exception e) {
+				// swallow any exception in attempting to get / set a match
```

Never swallow exceptions.  `e.printStackTrace()`